src/UI/Components/Diagram/stencil/helpers.test.ts

```diff
@@ -59,6 +63,23 @@ describe("createStencilElement", () => {
       textVerticalAnchor: "middle",
       fill: "#333333",
     });
+
+    //check difference with body fill for non-embedded
+    const nonEmbedded = createStencilElement("nonEmbedded", containerModel, {
+      attrOne: "test_value",
+      attrTwo: "other_test_value",
+    });
+
+    expect(nonEmbedded.attributes.attrs?.body).toStrictEqual({
```

Comment:
This sort of assertion is going to bite us later, why do we need to assert the exact shape? These are for the most hardCoded values we assign, so I'd assert only on the parts that are actually dynamic.